src/pages/history/[address].tsx

```diff
@@ -0,0 +1,78 @@
+import { useRouter } from 'next/router';
+import { useEffect } from 'react';
+import { isAddress } from 'viem';
+import { useForceInclusionHistory } from '@/hooks/useForceInclusionHistory';
+import HistoryList from '@/components/HistoryList';
+import { toast } from 'react-toastify';
+import Meta from '@/components/Meta';
+import { CiWarning } from 'react-icons/ci';
+import LoadingSpinner from '@/components/LoadingSpinner';
+
+export default function HistoryPage() {
+  const router = useRouter();
+  const { address } = router.query;
+  const { histories, isLoading, error } = useForceInclusionHistory(
+    address as string
+  );
+
+  useEffect(() => {
+    if (address && !isAddress(address as string)) {
+      toast.error('Invalid address');
+      router.push('/');
+    }
+  }, [address, router]);
+
+  return (
+    <>
+      <Meta />
+      <div className="min-h-[calc(100vh-64px)] w-full flex flex-col py-8">
+        <div className="flex-grow flex justify-center">
+          <div className="w-full max-w-2xl px-4">
+            <div className="bg-card rounded-lg shadow-lg p-6">
+              <div className="flex mb-6 flex-col gap-2">
+                <h1 className="text-2xl font-bold">History</h1>
+
+                <div className="flex items-center gap-2">
+                  <div className="text-sm text-gray-500">
+                    Force Inclusions in the last 5000 blocks:
```

Comment:
Just one last nit! 🤣🙏
```suggestion
                    Force Inclusions in the last 5000 L1 blocks:
```